feeds/wlan-ap/opensync/patches/44-sm_set_min_sig_neighbour_report.patch

```diff
@@ -0,0 +1,32 @@
+--- a/src/sm/src/sm_neighbor_report.c
++++ b/src/sm/src/sm_neighbor_report.c
+@@ -670,6 +670,29 @@ void sm_neighbor_stats_results(
+             continue;
+         }
+ 
++        /* Filter neighbors with low signal strength */
++        int min_sig;
++        if (scan_entry->type == RADIO_TYPE_2G) {
++            // 2.4G
++            min_sig = -83;
++        } else {
++            // 5G, 5GL, 5GU
++            min_sig = -85;
++        }
++
++        if (scan_entry->sig < min_sig) {
++            LOG(TRACE,
++                "Remove %s %s neighbor due to weak signal {bssid='%s' ssid='%s' rssi=%d min_rssi=%d chan=%d}\n",
++                radio_get_name_from_cfg(radio_cfg_ctx),
++                radio_get_scan_name_from_type(scan_type),
++                scan_entry->bssid,
++                scan_entry->ssid,
++                scan_entry->sig,
++                min_sig,
++                scan_entry->chan);
++            continue;
++        }
```

Comment:
Seems OK. But, cosmetically I would like to add the new filter to the existing one.

We already have a filter to remove entries with rssi_signal==0. We can add more filters to the same if case like below.

Current filter in the same function,
        /* Filter neighbor with 0 SNR */
        if (scan_entry->sig == 0) {

Add one more filter to the existing if case. Please also update the comment.
 if ((scan_entry->sig == 0) || (scan_entry->sig < min_sig))

Reply:
Fixed, also added some constants for the min values just so they aren't hardcoded directly in there. Probably overkill but hey...